x-pack/plugins/uptime/public/components/monitor/synthetics/step_detail/waterfall/data_formatting.ts

```diff
@@ -55,11 +61,32 @@ const getFriendlyTooltipValue = ({
   return `${label}: ${formatValueForDisplay(value)}ms`;
 };
 
-export const getSeriesAndDomain = (items: NetworkItems) => {
-  const getValueForOffset = (item: NetworkItem) => {
-    return item.requestSentTime;
-  };
+const getFriendlyMetaDataValue = ({ value, postFix }: { value?: number; postFix?: string }) => {
+  // value === -1 indicates timing data cannot be extracted
+  if (value === undefined || value === -1) {
+    return undefined;
+  }
+
+  let formattedValue = formatValueForDisplay(value);
+
+  if (postFix) {
```

Comment:
I could refactor this to add a default parameter of an empty string to post fix and then not have to have this if statement when postfix is undefined.